homeassistant/components/pjlink/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "pjlink",
   "name": "PJLink",
   "documentation": "https://www.home-assistant.io/integrations/pjlink",
-  "requirements": ["pypjlink2==1.2.0"],
+  "requirements": ["pypjlink2-rechner==1.2.2"],
```

Comment:
Please list all the steps you have taken to get your changes merged into the original package. Did you open a PR, did you reach out to the maintainer etc. We won't just change to forks because the odds that it gets unmaintained are very big.

Reply:
Rechner as well as well as several others, including myself, have opened pull requests over the past 6 months, there has been no communication or updates during that time, the last update to this code was on Jun 21, 2018.

Reply:
If a fork is made, it should be a real fork with a new name and version number. Forks that are one-off to fix something and just have the authors name appended to the original name tend to not stay maintained.

Reply:
makes sense, I'll be happy to sign on as a co-maintainer.